docs/source/visualization/TimeSeriesAnomalies.rst

```diff
@@ -345,6 +343,299 @@ the similar details
     </table>
     </div>
 
+Using MSTICPY - Seasonal-Trend decomposition using LOESS (STL)
+--------------------------------------------------------------
+
+In this case, we will use function msticpy function `timeseries_anomalies_stl` which leverages `STL` method from `statsmodels` API to decompose a time series into three components: trend, seasonal and residual. STL uses LOESS (locally estimated scatterplot smoothing) to extract smooths estimates of the three components. The key inputs into STL are:

```

Comment:
"we will use function msticpy function `timeseries_anomalies_stl` "
Think this should be "we will use the msticpy function `timeseries_anomalies_stl` "

Reply:
fixed errors.